.env.test

```diff
@@ -8,4 +8,7 @@ SECRET_KEY=secret-key
 TRELLO_API_KEY=trello-api-key
 TRELLO_TOKEN=trello-token
 TRELLO_BOARD_ID=trello-board-id
-TRELLO_ORGANIZATION_ID=trello-organization-id
\ No newline at end of file
+TRELLO_ORGANIZATION_ID=trello-organization-id
+
+DATABASE_CONNECTION_STRING=fakemongo.com:27017
+DATABASE_NAME=todo-app-database-test
```

Comment:
It would be good to remove the Trello keys here, and add the new DB fields to your `.env.template` file to make that really clear for future users